### PR TITLE
[NFC] Simplify WaveActiveMax test

### PR DESCRIPTION
**Pull request by llvm-beanz**
This was a test I had written to cover some edge cases, and the test is a bit awkward and buggy. This corrects the test to hopefully be more consistent across vendors.

### CODE REVIEW

**Review by farzonl — APPROVED** [review]

File: test/Feature/WaveOps/EdgeConditions/lit.cfg.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+if "EdgeConditions" in config.available_features:
+    config.unsupported = True
```
Comment (bogner) [inline test/Feature/WaveOps/EdgeConditions/lit.cfg.py]: This logic is exactly backwards
